src/reducers/existingThoughtDelete.js

```diff
@@ -26,6 +30,15 @@ export const existingThoughtDelete = (state, { thoughtsRanked, rank, showContext
   const contextEncoded = hashContext(context)
   const thoughtIndexNew = { ...state.thoughtIndex }
 
+  const oldPath = thoughtsRanked
```

Comment:
Rather than declaring a new variable `oldPath` which just serves as an alias, you can use `thoughtsRanked` directly. This will eliminate some complexity due to having a new variable for an existing concept.